<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Receiver</string>
     </attribute>
     <widget class="QPlainTextEdit" name="receiver_logDisplay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>751</width>
        <height>91</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Window</string>
      </property>
     </widget>
     <widget class="QPushButton" name="receiver_start">
      <property name="geometry">
       <rect>
        <x>758</x>
        <y>420</y>
        <width>100</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="receiver_freeze">
      <property name="geometry">
       <rect>
        <x>758</x>
        <y>450</y>
        <width>100</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Freeze</string>
      </property>
     </widget>
     <widget class="QPushButton" name="receiver_stop">
      <property name="geometry">
       <rect>
        <x>758</x>
        <y>480</y>
        <width>100</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="receiver_clock">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>50</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
      <property name="suffix">
       <string notr="true"> Hz</string>
      </property>
      <property name="maximum">
       <number>100000000</number>
      </property>
      <property name="value">
       <number>80000000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="receiver_setDeviceConfiguration">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>170</y>
        <width>219</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Device Configuration</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="receiver_nominalBitRate">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>90</y>
        <width>170</width>
        <height>27</height>
       </rect>
      </property>
      <property name="suffix">
       <string> bps</string>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>1000000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="receiver_dataBitRate">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>120</y>
        <width>170</width>
        <height>27</height>
       </rect>
      </property>
      <property name="suffix">
       <string> bps</string>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="value">
       <number>2000000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="receiver_getDeviceConfiguration">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>420</y>
        <width>219</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Device Configuration</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Transmitter</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
